### DISCUSSION TITLE
what's the fullscreen layout function?

### DISCUSSION THREAD

**Discussion by ztoiax** [qtile/qtile#2407] [Q&A]
I want to like that:
```
Key([mod], "w", lazy.layout.Max(), desc="switch to fullscreen layout"),
```

**Comment by ramnes** [comment]
If I understand correctly what you're looking for, I think the only thing we have at the moment is `lazy.to_layout_index`.

http://docs.qtile.org/en/latest/manual/ref/commands.html?highlight=to_layout#libqtile.core.manager.Qtile.cmd_to_layout_index
